espanso-config/src/legacy/config.rs

```diff
@@ -760,7 +760,7 @@ impl fmt::Display for ConfigLoadError {
             ConfigLoadError::InvalidYAML(path, e) => write!(f, "Error parsing YAML file '{}', invalid syntax: {}", path.to_str().unwrap_or_default(), e),
             ConfigLoadError::InvalidConfigDirectory =>  write!(f, "Invalid config directory"),
             ConfigLoadError::InvalidParameter(path) =>  write!(f, "Invalid parameter in '{}', use of reserved parameters in used defined configs is not permitted", path.to_str().unwrap_or_default()),
-            ConfigLoadError::NameDuplicate(path) =>  write!(f, "Found duplicate 'name' in '{}', please use different names", path.to_str().unwrap_or_default()),
+            ConfigLoadError::NameDuplicate(path) =>  write!(f, "Found duplicate filename, please use different names for your config files."),
```

Comment:
The `path` was not helpful in the resulting message. Upon deleting the package, the same error with a different package path appeared.

Thus, can either the _file_ paths of both duplicates be listed here, maybe? I'd be happy to let someone take over this PR, since I'm not very experienced in Rust.